pkg/noderesourcetopology/README.md

```diff
@@ -98,6 +98,24 @@ profiles:
       cacheResyncPeriodSeconds: 5
 ```
 
+#### ScoringStrategy
+
+Topology-aware scheduler supports four scoring strategies. You can set a strategy via SchedulerConfigConfiguration, by setting `scoringStrategy` option. There are four supported strategies:
```

Comment:
I'd write: "The topology-aware scheduler supports four scoring strategies. You can set a strategy via SchedulerConfigConfiguration, by setting the `scoringStrategy` option."

Reply:
👍